.github/workflows/release.yml

```diff
@@ -0,0 +1,213 @@
+name: Tag a new release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: |
+          The release version in <Major>.<minor> format.
+          0.0 means new minor version on the latest major version.
+        default: '0.0'
+        type: string
+      dry-run:
+        description: Do not push anything
+        default: true
+        type: boolean
+
+jobs:
+  determine-version:
+    runs-on: ubuntu-24.04
+
+    outputs:
+      major: ${{ steps.final-values.outputs.major }}
+      minor: ${{ steps.final-values.outputs.minor }}
+      patch: ${{ steps.patch-version.outputs.value || '0' }}
+      release-type: ${{ steps.final-values.outputs.type }}
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: false
+          fetch-depth: 0
+
+      - name: Parse required release
+        id: required-release
+        run: |
+          if [[ "${{ inputs.version }}" =~ ^([[:digit:]]+)\.([[:digit:]]+)$ ]]; then
+            echo "major=${BASH_REMATCH[1]}" >> "$GITHUB_OUTPUT"
+            echo "minor=${BASH_REMATCH[2]}" >> "$GITHUB_OUTPUT"
+          else
+            echo >&2 "Invalid version ${{ inputs.version }}"
```

Comment:
```suggestion
            echo >&2 "Invalid version ${{ inputs.version }}. The expected format is x.y"
```

Reply:
If it is alright with you, I'll do a small variation of this to make it more clear, something like `The expected format is <Major>.<minor>`

Reply:
That's fine.